docs/onboarding.md

```diff
@@ -21,6 +21,7 @@ checklist. Replace {TockDeveloper} with the name of the onboarding developer.
 - [ ] {TockDeveloper} has been invited to #tock-dev in Slack.
 - [ ] {TockDeveloper} is familiar with the [deployment process of
   Tock](https://github.com/18f/tock/tree/master/docs/deployment-process.md).
+- [ ] {TockDeveloper} has access to `GSATETockLibrary` and `GSATEUtilitiesLibrary` scripts.
```

Comment:
Can we explain how someone would get access to these scripts? I had Angela look through the Google Apps Script UI interface via search and asked her to grant me permission. I think a line along the thread of “ask the person who previously had access or the product owner of Tock for access”.

Reply:
The instructions are for the onboarder to load into a GH issue, but I have added a line to clarify that they have to do that for the new, onboarding developer. .